homeassistant/components/input_select/strings.json

```diff
@@ -16,5 +16,59 @@
         }
       }
     }
+  },
+  "services": {
```

Comment:
I think pretty much all of these we can get from `select` entity component #96411

Reply:
Probably same for all the others here that have a "matching" entity component

Reply:
We will de-dupe more later using scripts.